Updating extension-readme for CloudWatch Logs functionality (#1253)
aws/aws-toolkit-vscode: extension-readme.md

---

## <a id="cloudwatchlogs"></a>Amazon CloudWatch Logs

The AWS Toolkit provides support for [Amazon CloudWatch Logs](https://aws.amazon.com/cloudwatch/). Using the [AWS Explorer](#ui-components-aws-expl) of the Toolkit, you can perform the following operations on Log Groups:

-   List CloudWatch Log Groups

>     -   View Log Streams for a Log Group

Reviewer: nit: i can see why you made indented this, but maybe it's more distracting than helpful

> Viewing a Log Stream will immediately load the most recent 10,000 lines or 1 MB of data (whichever is smaller), as well as their timestamps. From this view, users will have access to the following actions:

Reviewer: ```suggestion
Viewing a Log Stream will immediately load the most recent 10,000 lines or 1 MB of data (whichever is smaller) and their timestamps. From this view, users can access the following actions:
```
